db/queries/recipe/list_recipes_with_query.sql

```diff
@@ -1,24 +1,48 @@
+--Used when there is not a featured recipe
 -- name: ListRecipesWithQuery :many
-WITH rankings AS (
+WITH LatesDateRecipe AS 
+ (     
+ SELECT 
+    ROW_NUMBER() OVER (PARTITION BY recipe_id ORDER BY publish_date DESC) AS row_num,
+    recipe_id,
+    publish_date,
+    recipe_description,
+    title,
+    id
```

Comment:
Can we cut this down to just `row_num`, `recipe_id`, `recipe_description` and `title`?